src/app/components/game-center/game-center.component.ts

```diff
@@ -30,14 +30,14 @@ export class GameCenterComponent {
         break;
       }
       default: {
-        this.word += letter;
+        if (this.word.length < 5) this.word += letter;
+        this.lenghtWord = this.word.length < 5;
```

Comment:
```suggestion
        if (this.word.length < 5) this.word += letter;
```

Con este if, nunca vas a tener la palabra con más de 5 caracteres, con lo que la función de disable tiene que ser === 5